src/timings/Timings.php

```diff
@@ -299,4 +306,43 @@ public static function getEventHandlerTimings(string $event, string $handlerName
 
 		return self::$eventHandlers[$event][$handlerName];
 	}
+
+	public static function getAsyncTasksProgressUpdateTimings(AsyncTask $task) : TimingsHandler{
+		$taskClass = get_class($task);
+		if(!isset(self::$asyncTasksProgressUpdate[$taskClass])){
+			self::$asyncTasksProgressUpdate[$taskClass] = new TimingsHandler(
+				"AsyncTask Progress Update - " . $taskClass,
+				self::$schedulerAsync,
```

Comment:
This is problematic for offline/unit testing, as the timer may not be initialized